docs/components/connectors/out-of-the-box-connectors/rest.md

```diff
@@ -41,8 +42,9 @@ We advise you to keep your **Password** safe and avoid exposing it in the BPMN `
 
 Select the **REST Connector** and fill out the following properties under the **Authentication** section:
 
-1. Set **Username** (i.e. `YOUR_USERNAME`).
-2. Set **Password** to the secret you created (i.e. `secrets.REST_BASIC_AUTH_SECRET`).
+1. Click **Basic Auth** in the **Authentication** section.
+2. Set **Username** (i.e. `YOUR_USERNAME`).
+3. Set **Password** to the secret you created (i.e. `secrets.REST_BASIC_AUTH_SECRET`).
 
 ![rest Connector basic auth](../img/connectors-rest-basic-auth.png)
```

Comment:
Should there be a secrets.YOUR_USERNAME here? if not should we use <YOUR_USERNAME>?